src/main/java/org/httpserver/config/ConfigurationManager.java

```diff
@@ -0,0 +1,66 @@
+package org.httpserver.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.httpserver.util.Json;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ConfigurationManager {
+    private static ConfigurationManager myConfigurationManager;
+    private static Configuration myCurrentConfiguration;
+    private ConfigurationManager() {
+    }
+    public static ConfigurationManager getInstance(){
+        if (myConfigurationManager == null)
+            myConfigurationManager = new ConfigurationManager();
+        return myConfigurationManager;
+    }
+
+    /**
+     *
+     * Used to load a configuration file by the path provided
+     */
+    public void loadConfirurationFile(String filePath) {
+        FileReader fileReader = null;
+        try {
+            fileReader = new FileReader(filePath);
+        } catch (FileNotFoundException e) {
+            throw new HttpConfigurationException(e);
+        }
+        StringBuffer sb = new StringBuffer();
+        int i;
+        try {
+            while ( ( i = fileReader.read()) != -1) {
+                sb.append((char)i);
+            }
+        } catch (IOException e) {
+            throw new HttpConfigurationException(e);
+        }
+        JsonNode conf = null;
+        try {
+            conf = Json.parse(sb.toString());
```

Comment:
Now imagine you would have to call `Json.parse(...)` anywhere else 10 times. 
You would write this `try-catch` block 10 times in a different places. 

I suggest moving this `try-catch` block into the `Json.parse(...)` method. 
And don't forget to change the `HttpConfigurationException` message from a one, that is specific about parsing a configuration file to something more generic.